src/raft.rs

```diff
@@ -124,6 +124,14 @@ pub struct Config {
     /// rejoins the cluster.
     pub pre_vote: bool,
 
+    /// The range of election timeout. In some cases, we hope some nodes has less possibility
+    /// to become leader. This configuration ensures that the randomized election_timeout
+    /// will always be suit in [min_election_tick, max_election_tick).
+    /// If it is None, then election_tick will be chosen.
+    pub min_election_tick: Option<usize>,
```

Comment:
I think you can remove Option here, you can set min to election timeout and max to 2 x election timeout explicitly if they are 0 at initialization.